[Bulk] Validation Fixes (#254)
MicrosoftDocs/bingmaps-docs: BingMaps/rest-services/directly-accessing-the-bing-maps-tiles.md

---
title: "Directly Accessing the Bing Maps Tiles | Microsoft Docs"

> description: "This article outlines direct access the Bing Map tiles and how to add Bing Maps as a tile layer in a non-Bing Maps control."

Reviewer: This article outlines direct access to the Bing Map tiles and how to add Bing Maps as a tile layer in a non-Bing Map control.

> description: "This article outlines direct access to the Bing Map tiles and how to add Bing Maps as a tile layer in a non-Bing Maps control."

Reviewer 2: ```suggestion
description: Learn how to access the Bing Map tiles directly to add Bing Maps as a tile layer in a non-Bing Maps control."
```
